spring/src/main/java/org/axonframework/spring/config/AbstractAnnotationHandlerBeanPostProcessor.java

```diff
@@ -51,6 +51,7 @@
  * @author Allard Buijze
  * @since 0.4
  */
+@Deprecated
```

Comment:
This change warrants an adjustment in the JavaDoc describing why it's deprecated and what (if any) replaces it.